### PR TITLE
Add test for finding nested dir at path.basename(cwd)

### PR DESCRIPTION
**Pull request by coreyfarrell**
@sholladay this adds passing tests to your branch for edge cases including searching for an absolute path (existing and not).  I'm posting this PR to your fork to avoid conflicting with sindresorhus/find-up#28.

CC @sindresorhus

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -220,6 +238,46 @@ test('sync (ancestor directory, custom cwd)', t => {
...
+	t.is(filePath, absolute.barDir);
+});
+
+test('sync (absolute directory)', t => {
+	const filePath = findUp.sync(absolute.barDir);
+
+	t.is(filePath, absolute.barDir);
+});
+
+test('async (not found, absolute directory)', async t => {
```
Comment (sholladay) [inline test.js]: I think you meant to write "absolute file"? (Same with the sync test.)

Comment (coreyfarrell) [reply test.js]: fixed.

**Review by sindresorhus — APPROVED** [review]

**Comment by sholladay** [comment]
Thanks, tests are always appreciated. :)
